<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="listWidget">
        <item>
         <property name="text">
          <string>北京</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/new/images/images/circle_triangle_right.png</normaloff>:/new/images/images/circle_triangle_right.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>上海</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/new/images/images/circle_triangle_right.png</normaloff>:/new/images/images/circle_triangle_right.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>张家口</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/new/images/images/circle_triangle_right.png</normaloff>:/new/images/images/circle_triangle_right.png</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="text">
           <string>delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="deleteAllButton">
        <property name="text">
         <string>delete All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showDirButton">
        <property name="text">
         <string>showdir</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
